c/driver/postgresql/statement.cc

```diff
@@ -426,6 +430,23 @@ struct BindStream {
               std::memcpy(param_values[col], &value, sizeof(int64_t));
               break;
             }
+            case ArrowType::NANOARROW_TYPE_INTERVAL_MONTH_DAY_NANO: {
+              const auto buf =
```

Comment:
One thing that is missing from https://github.com/apache/arrow-nanoarrow/pull/258/files is a `ArrowArrayViewGetIntervalUnsafe` function, which would help both here and in the test